cmd/generate/main.go

```diff
@@ -9,19 +9,44 @@ package main
 
 import (
 	"flag"
+	"fmt"
 	"log"
 	"os"
 
 	"github.com/nginxinc/nginx-go-crossplane/internal/generator"
 )
 
+type stringSliceFlag []string
+
+func (s *stringSliceFlag) String() string {
+	return fmt.Sprintf("%v", *s)
+}
+
+func (s *stringSliceFlag) Set(value string) error {
+	*s = append(*s, value)
+	return nil
+}
+
 func main() {
 	var (
 		sourceCodePath = flag.String("src-path", "",
-			"the path of source code your want to generate support from, it can be either a file or a directory. (required)")
+			"The path of source code your want to generate support from, it can be either a file or a directory. (required)")
+		filter   stringSliceFlag
+		override stringSliceFlag
 	)
+	flag.Var(&filter, "filter",
+		"A list of strings specifying the directives to include in the output. "+
+			"If not provided, all directives in the src-path will be included.\n"+
+			"An example is: -filter directive1 -filter directive2...(optional)")
+	flag.Var(&override, "override",
+		"A list of strings, used to override the output. "+
+			"It should follow the format:{directive:mask1,mask2,...}\n"+
+			"An example is -override=log_format:ngxHTTPMainConf|ngxConf2More,ngxStreamMainConf|ngxConf2More\n"+
```

Comment:
Why does the caller need to provide `\n`?

Reply:
Originally I thought this is just for `go run main.go --help`, and the new line provides a better reading experience. I will move `\n` to the begin of next line

Reply:
Fixed: split "\n" out